roles/cloudman-boot/tasks/storage.yaml

```diff
@@ -24,3 +24,20 @@
     --set params.s3-secret-access-key="{{ (cm_initial_cluster_data | b64decode | from_yaml).cloud_config.credentials.aws_secret_key }}"
   when: kube_cloud_provider == "aws"
   ignore_errors: true
+
+- name: Helm install RClone for OpenStack Swift
+  command: >
+    /usr/local/bin/helm install rclone-csi wunderio/csi-rclone
+    --namespace "{{ cm_namespace_name }}"
+    --set storageClass.name="rclone"
+    --set params.remote="swift"
+    --set params.remotePath="{{ rancher_server | replace('.', '-') }}-gvl-data"
+    --set params.swift-user="{{ (cm_initial_cluster_data | b64decode | from_yaml).cloud_config.credentials.os_username }}"
+    --set params.swift-key="{{ (cm_initial_cluster_data | b64decode | from_yaml).cloud_config.credentials.os_password }}"
+    --set params.swift-auth="{{ (cm_initial_cluster_data | b64decode | from_yaml).cloud_config.target.target_zone.cloud.auth_url }}"
+    --set params.swift-domain="{{ (cm_initial_cluster_data | b64decode | from_yaml).cloud_config.credentials.os_project_domain_name }}"
```

Comment:
Some openstack providers now give only the os_project_domain_id (hence the corresponding change in cloudlaunch). Is there an option to provide the id in addition to the name?

Reply:
https://rclone.org/swift/#swift-tenant-domain
They explicitly only mention name not id, but I haven't tried it manually though